document/CoreFormat.mdk

```diff
@@ -319,6 +326,15 @@ Analysis tools that are configurable to produce hashes with a variety of commonl
 will interoperate most easily with such systems.
 ~
 
+### `tags` property { #fileReference-tags }
+
+A `fileReference` object may contain a property named `tags`, whose value is an array of zero or more arbitrary strings.
```

Comment:
Good point. I will DRY this out by defining a "tag list" as I did "property bag", and state the requirement there.